### ISSUE TITLE
Rename chat conversations

### ISSUE THREAD

**Issue by Kerollmops** [zeozeozeo/ellama#17]
It would be great to rename chat conversations and even be able to press a button to ask the AI to do so. I tried to ask the AI but it is always too wordy. It would be preferable to set up a conversation just for that.

**Comment by zeozeozeo** [comment]
> Sorry, I wasn't clear I think. I was talking about the chat conversation titles.

Oh, you mean using an LLM to generate a short summary of the chat? Yeah, that would be nice, but it also would be quite hard to implement considering that Ollama cannot generate completions using multiple models at once, and some models might give unexpected output... I would prefer the title to be generated automatically instead of it being a user action
